src/main/java/com/smartcodeltd/jenkinsci/plugins/buildmonitor/viewmodel/BuildView.java

```diff
@@ -23,17 +24,18 @@
     private final RelativeLocation parentJobLocation;
     private final Date systemTime;
     private final BuildAugmentor augmentor;
+    private final Config config;
 
-    public static BuildView of(Run<?, ?> build) {
-        return new BuildView(build, new BuildAugmentor(), RelativeLocation.of(build.getParent()), new Date());
+    public static BuildView of(Run<?, ?> build, Config config) {
+        return new BuildView(build, config, new BuildAugmentor(), RelativeLocation.of(build.getParent()), new Date());
```

Comment:
Short answer is yes. As an example, multimodule needs to change the definition of BuildView.isRunning based on config and aug. I'm not sure of the best way (different Story) but they do both play a part.